egs/wsj/s5/steps/nnet3/multilingual/allocate_multilingual_examples.py

```diff
@@ -45,6 +45,8 @@
     for validation examples and "combine." for examples used for model
     combination.
 
+    For chain training egs, the --egs-prefix option should be "cegs."
```

Comment:
If you merge Hossein's PR with this PR, there will likely be things you need to do about this script.